core/templates/dev/head/profile/signup.html

```diff
@@ -59,18 +54,17 @@ <h2 class="oppia-signup-page-title">Complete Your Registration</h2>
               <div class="radio">
                 <label>
                   <input type="radio" ng-model="canReceiveEmailUpdates" value="yes" name="canReceiveEmailUpdates" ng-change="onSelectEmailPreference()">
-                  Send me news and updates about the site
+                  <span translate="I18N_SIGNUP_SEND_ME_NEWS"></span>
                 </label>
                 <label>
                   <input type="radio" ng-model="canReceiveEmailUpdates" value="no" name="canReceiveEmailUpdates" ng-change="onSelectEmailPreference()">
-                  Do not send these emails
+                  <span translate="I18N_SIGNUP_DO_NOT_SEND_EMAILS"></span>
                 </label>
-                <span class="help-block" style="font-size: smaller">
-                  You can change this setting at any time from your Preferences page.
+                <span class="help-block" style="font-size: smaller" translate="I18N_SIGNUP_EMAIL_PREFERENCES_EXPLAIN">
                 </span>
               </div>
               <div style="height: 18px; width: 550px;">
-                <span style="color: red; font-size: 0.8em;" aria-live="assertive" ng-show="blurredAtLeastOnce">
+                <span style="color: red; font-size: 0.8em;" aria-live="assertive" ng-show="blurredAtLeastOnce" translate>
```

Comment:
Without arguments, the attribute translate will take as translation id the content of the DOM element. In this case, <[emailPreferencesWarningText]>